README.md

```diff
@@ -70,6 +70,12 @@ us single bullet fire weapons.
 9. Build the VS project
 10. The executables are placed at `../game/hl2.exe` for the client and at `../game/srcds.exe` for the server. Note: this path is outside the repository.
 
+### Building on Linux (To the Make File)
+**DISCLAIMER:** Make sure you have your distro's build essentials, gcc and g++ multilibs installed. This will varry from distro to distro, so I can't include everything here.
+1. Install gcc and g++ multilibs, build essentials. On arch I used the command `sudo pacman -Syu ib32-gcc-libs gcc-libs`. For Ubuntu, according to the Valve Wiki for Source SDK 2013, do `sudo apt-get install build-essential` and `sudo apt-get install gcc-multilib g++-multilib`

```

Comment:
missing first letter of "lib32-gcc-libs" 
you could combine these 2 apt-get commands together